packages/docs/package.json

```diff
@@ -2,5 +2,11 @@
   "name": "docs",
   "version": "0.0.3",
   "private": true,
-  "license": "LicenseRef-LICENSE"
+  "license": "LicenseRef-LICENSE",
+  "scripts": {
+    "dev": "mintlify dev"
+  },
+  "devDependencies": {
```

Comment:
Could you remove this?

I used to include mintlify as a dev dep but a bunch of people were having trouble installing/building mintlify on their machines. So, when they'd run `yarn` or `yarn install`, it'd fail because mintlify would fail to install, even though they'd be working on exclusively e.g. the dashboard. I think it has to do with one of mintlify's native dependencies.

We already document in the docs README that mintlify should be installed globally.

Reply:
Sure Max 🙂